packages/wonder-blocks-core/generated-snapshot.test.js

```diff
@@ -56,16 +56,10 @@ describe("wonder-blocks-core", () => {
                 backgroundColor: "pink",
             },
         });
-
+        
         const StyledInput = addStyle("input", styles.input);
-
-        const example = (
-            <StyledInput
-                style={styles.pink}
-                type="text"
-                placeholder="hello, world"
-            />
-        );
+        
+        const example = <StyledInput style={styles.pink} type="text" placeholder="hello, world"/>;
```

Comment:
Why the change in formatting?

Reply:
Can you run `npm run pretty-quick` manually?

Reply:
pretty-quick doesn't do anything if there are no changes. I ended up having to run `./node_modules/.bin/prettier --write packages/wonder-blocks-*/generated-snapshot.test.js` - but that changed even more files (I think prettier isn't getting run against these files for some reason).